README.md

```diff
@@ -49,6 +49,12 @@ cf set-env $YOUR_APP_NAME DD_LOGS_CONFIG_TCP_FORWARD_PORT 10514
 cf set-env $YOUR_APP_NAME DISABLE_STD_LOG_COLLECTION true
 ```
 
+To collect metrics with DogStatsD only, it is possible to disable metric collection in the puppy agent by adding:
```

Comment:
How else would you collect metrics here other than dogstatsd?

Reply:
The puppy agent runs some core checks by default